modules/migration-downloading-logs.adoc

```diff
@@ -11,14 +11,16 @@ You can download the `Velero`, `Restic`, and `MigrationController` pod logs in t
 
 .Procedure
 
-. Log in to the {mtc-short} console.
-. Click *Plans* to view the list of migration plans.
-. Click the *Options* menu {kebab} of a specific migration plan and select *Logs*.
-. Click *Download Logs* to download the `MigrationController`, `Velero`, and `Restic` pod logs for all clusters.
-. To download a single log, specify the following options:
+. In the {mtc-short} console, click *Migration plans* to view the list of migration plans.
+. Click the Options menu {kebab} of a specific migration plan and select *Logs*.
```

Comment:
I think that Options should be bolded based on [this part](https://redhat-documentation.github.io/supplementary-style-guide/#graphical-interfaces) of the supplementary style guide.

Reply:
@kalexand-rh 
I was told by @bergerhoffer (I think) that UI element names should only be bolded if they appear in the UI. In this case,  `Options` would have to appear as a tooltip. However, this UI has no tooltip, so `Option` never appears.

Does this requirement to 'bold' UI elements include elements that are not explicitly named in the UI?

Reply:
The IBM Style Guide also says:
> Bold on icon labels and tooltip names

Reply:
That doesn't answer my question. This is a case where there is no icon label or tooltip name.

Reply:
In this case, there is no icon label or tooltip name. But if Red Hat Supp. style trumps IBM, then all UI element names are bolded. So that's what I'll do.

Reply:
The Red Hat Supplementary Style is meant to clarify and trump the IBM Style.

Reply:
Yeah it's not super clear, but if the word "Options" doesn't actually appear in the GUI, I'm not sure if it should be bolded. I see it both ways in the repo now, with more instances not bolding. And it if doesn't actually appear in the GUI, it actuall should probably not be capitalized.

I feel like there has been discussion on what to call the kebab menu icon (which I'm assuming landed on options menu), but I'm not sure if whoever decided on that has any thoughts on if it should be capitalized and/or styled as a GUI item. I don't know off the top of my head who was involved in that, but I think it was a larger CCS discussion. 

Either way, I think this should be added to the supplementary style guide. I've opened it as a discussion here: https://github.com/redhat-documentation/supplementary-style-guide/issues/49

But this PR is fine as you have it now, having the icon after it makes it clear enough to users, no matter what we actually call it or how it's styled.